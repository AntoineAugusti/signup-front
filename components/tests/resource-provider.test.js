import ReactTestRenderer from 'react-test-renderer'
import ResourceProvider from '../resource-provider'
import RESOURCE_PROVIDERS from '../../mock/data/resource-providers'
const RESOURCE_PROVIDER = RESOURCE_PROVIDERS[0]

describe('components | DataSet', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<ResourceProvider resourceProvider={RESOURCE_PROVIDER} />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
