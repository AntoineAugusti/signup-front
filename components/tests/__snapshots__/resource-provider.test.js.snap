// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | DataSet render should match snapshot 1`] = `
<section
  className="section-grey"
>
  <div>
    <div
      className="container"
    >
      <h2
        className="section__title"
      >
        Données issues de la 
        <abbr
          title="Direction Générale des Finances Publiques"
        >
          DGFIP
        </abbr>
      </h2>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque nulla lorem, maximus vel nisl in, finibus aliquam nunc. Quisque malesuada nisi nec mi interdum rutrum. Maecenas vel magna sit amet nulla volutpat finibus non nec ex. Pellentesque eu lectus tortor. Donec semper malesuada nisl eu elementum. Nulla placerat nisl ut massa luctus consectetur. Praesent varius sit amet sapien at suscipit. Pellentesque bibendum iaculis turpis, at lobortis sem sodales id. Nulla vitae auctor turpis, id tristique elit. Vestibulum euismod dolor dictum nulla maximus, vitae maximus nunc efficitur.
      </p>
      <div
        className="panel"
      >
        <div
          className="panel__header"
        >
          <h3>
            Avis d'imposition
          </h3>
          <small
            className="panel__header-extra"
          >
            DGFIP
          </small>
        </div>
        <div>
          <p>
            Données issues du service svair de la DGFIP
          </p>
          <div
            className="form__group"
          >
            <label>
              Qui utilise ces données ?
            </label>
            <ul
              className="tag-list"
            >
              <li
                className="tag"
              >
                <a
                  href={undefined}
                >
                  BourseSCO
                </a>
              </li>
              <li
                className="tag"
              >
                <a
                  href={undefined}
                >
                  Lyon : Mon compte
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="panel__actions"
        >
          <a
            className="button button-secondary"
            href=""
            onClick={[Function]}
          >
            Essayez-moi !
          </a>
          <button
            className="button"
            id="dgfip_avis_imposition_submit_button"
            name="dgfip_avis_imposition_button"
            type="submit"
          >
            Ajouter à ma sélection
          </button>
        </div>
      </div>
    </div>
  </div>
  <div>
    <img
      alt={null}
      src={null}
    />
  </div>
</section>
`;
